### PR TITLE
Feature/ds 7801 websocket1

### PR DESCRIPTION
**Pull request by agevorgy**
basic socket connection

### CODE REVIEW

**Review by cpm5041 — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -13,9 +13,20 @@ var app = express();
...
 var database = process.env.DATABASE;
 
+var http = require('http');
+var server = http.createServer(app);
+var io = require('socket.io').listen(server);
+
+io.on('connection', (socket) => {
+  console.log('Connected');
+  socket.on('join', function(data) {
+    console.log(data);
```
Comment (juanflo) [inline app.js:22]: Shouldn't we broadcast to all clients?

File: app.js [hunk 2]
```diff
@@ -13,9 +13,20 @@ var app = express();
 var port = process.env.PORT || 3005;
 var database = process.env.DATABASE;
 
+var http = require('http');
```
Comment (juanflo) [inline app.js]: `express().listen()` returns an http server so you don't need to import it.

**Review by juanflo — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (agevorgy) [reply app.js:22]: yes, but this is just the basic connection. we're working on broadcasting to all the clients now

File: package.json [hunk 3]
```diff
@@ -18,7 +18,8 @@
     "mongoose": "^5.1.5",
     "morgan": "~1.9.0",
     "short-id": "0.1.0-1",
-    "shortid": "^2.2.8"
+    "shortid": "^2.2.8",
```
Comment (juanflo) [inline package.json]: I'm assuming you are using this one. Can you remove `short-id` above it. `yarn remove short-id`.

**Review by juanflo — CHANGES_REQUESTED** [review]

**Review by juanflo — APPROVED** [review]

**Pull request merged**